src/lesson6_task1/CargoTransport.java

```diff
@@ -0,0 +1,29 @@
+package lesson6_task1;
+
+public class CargoTransport extends LandTransport {
+    float loadingCapacity;
+
+    //конструктор
+    public CargoTransport(int powerHours, int maxSpeed, float weight, String brand, int countOfWheels, float fuelConsumption,
+                          float loadingCapacity) {
+        super(powerHours, maxSpeed, brand, weight, countOfWheels, fuelConsumption);
+        this.loadingCapacity = loadingCapacity;
+    }
+
+    //возвращает строку с описанием всех характеристик объекта
+    @Override
+    public String getDescription() {
+        this.technicalChar = "Марка " + this.brand + ":\n-Мощность " + this.powerHours + " лошадиных сил;\n-Мощность " + getPowerKw() + " кВ;\n" +
+                "-Максимальная скорость " + this.maxSpeed + " км/ч;\n-Масса " + this.weight + " кг;\n" +
+                "-Количество колёс " + this.countOfWheels + ";\n-Расход топлива " + this.fuelConsumption + " л/100км;\n" +
+                "-Грузоподъёмность " + this.loadingCapacity + " т.\n";
+        return this.technicalChar;
+    }
+
+    //проверить можно ли загрузить в него xxx груза
+    public void checkCargoWeight() {
+        float cargoWeight = getFloat();
+        if (cargoWeight < this.loadingCapacity) System.out.println("Грузовик загружен.");
```

Comment:
```suggestion
        if (cargoWeight < this.loadingCapacity) {
             System.out.println("Грузовик загружен.");ntln("Грузовик загружен.");
       }         
```
@Nadya101 Все таки лучше отделять фигурными скобками. Без них лучше в Kotlin, ошибся немного

Reply:
- [ ] Сделано)